### PR TITLE
feat: Add Command loading and sending it to the Engine

### PR DESCRIPTION
**Pull request by petervdmeer**
**Feature:** This PR implements the command loading and sending feature from #4370

## Feature Details
This PR adds a function to the Command class to load commands from datafiles and adds a function in the Engine to use a given command in the active frame.

This is part of the automatic-testing PR #4370 and can also be used for mission scripted commands as suggested in PR #4720.
The automatic testing PR was getting quite large, so splitting this off so that it can be reviewed more easily.

## UI Screenshots
N/A

## Usage Examples
See #4370 for usage of this feature.

## Testing Done
Tested multiple times as part of #4370.

## Performance Impact
No performance impact expected. This functionality would only be used during automated testing and/or during scripted missions and would typically be used only once per frame during such usage.

### CODE REVIEW

**Comment by petervdmeer** [comment]
Closing, to be handled as part of #4370.

**Comment by petervdmeer** [comment]
Re-opened; #4370 is not ready for merging yet, this PR might be merged ahead (to make #4370 smaller).

File: source/Command.cpp [hunk 1]
```diff
@@ -221,6 +221,48 @@ bool Command::HasConflict() const
...
+			{"gather", Command::GATHER},
+			{"hold", Command::HOLD},
+			{"ammo", Command::AMMO}
+		};
+		
+		auto it = lookup.find(node.Token(i));
+		if(it != lookup.end())
+			Set(it->second);
+		else
+			node.PrintTrace("Skipping unrecognized command:");
```
Comment (tehhowch) [inline source/Command.cpp:260]: Seems like the input node can be e.g. `<somekey> "primary" "secondary" "tertiary" "ammo"`, so we should also flag the particular skipped item
```suggestion
			node.PrintTrace("Skipping unrecognized command \"" + node.Token(i) + "\":");
```

**Comment by petervdmeer** [comment]
> Looks much better without the huge if-else if chain

Yes, it indeed looks better.

**Pull request merged**